lib/modules/manager/nuget/extract.ts

```diff
@@ -52,13 +51,7 @@ function extractDepsFromXml(xmlNode: XmlDocument): NugetPackageDependency[] {
       const dep = getDep(child.val, true);
 
       if (is.nonEmptyStringAndNotWhitespace(dep.depName)) {
-        results.push({
-          datasource: DockerDatasource.id,
-          depType: 'docker',
-          depName: dep.depName,
-          currentValue: dep.currentValue,
-          currentDigest: dep.currentDigest,
-        });
+        results.push(dep);
```

Comment:
```suggestion
        results.push({ ...dep, depType: 'docker' });
```

we should retain the `depType`, otherwise we probably break other users